Java-Project-Management/src/ngo_2024/Anstalld.java

```diff
@@ -148,18 +167,47 @@ public ArrayList<HashMap<String, String>> allaAnstallda()
         return allaAnstallda;
     }
     
-    public HashMap<String, String> getEnAnstalld()
+    /**
+     * Metod för att hämta ut en Anställd via namn.
+     * @param namn
+     * @return 
+     */
+    
+    public HashMap<String, String> getEnAnstalldViaNamn(String namn)
+    {
+        HashMap<String, String> enAnstalld = new HashMap<>();
+        try
+        {
+            String sqlfråga = "SELECT * FROM anstalld WHERE namn = '" + namn + "'";
```

Comment:
Funkar denna select-fråga? namn finns väl inte som kollumn i anstalld, får väl koppla mot fornamn = ' ' AND efternamn =' '?

Reply:
Bortsett detta ser det bra ut, jag har börjat sätta catch-felmeddelandena i System.out.println istället för i pop-up-rutan, för resonerar att användaren behöver inte java-texten bara allmänt "inte uppdaterad" eller "fel format" eller vad det nu skulle kunna vara

Reply:
Nej, den ska inte vara med.